### PR TITLE
Add load-test case for PRs

### LINKED ISSUE #403: :star: Add a quest test in load mode
We need to add a test where we run the [quest](https://github.com/parseablehq/quest) test in load mode against each pr.

### PR DESCRIPTION
**Pull request by Sajiyah-Salat**
Fixes #403 

### Description

*goal of this PR*: 
This pr introdues load testing capabilities to enhance the performance of the project. 
Changes: 
Configured github workflow to run load test against each pr. 

This PR has:
- [ ] been tested to ensure log ingestion and log query works.
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added documentation for new or modified features or behaviors.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -70,3 +70,20 @@ services:
...
         max_attempts: 3
+        
+  quest-load:
+    image: ghcr.io/parseablehq/quest:main
+    command: ["load", "http://parseable:8000", "parseableadmin", "parseableadmin", "20"]
+    networks:
+      - parseable-internal
+    depends_on:
+      quest-smoke: 
+        condition: service_completed_ successfully
```
Comment (nitisht) [inline docker-compose.yaml:81]: extra space here needs to be removed

**Review by nitisht — CHANGES_REQUESTED** [review]

File: docker-compose.yaml [hunk 2]
```diff
@@ -70,3 +70,20 @@ services:
...
+        
+  quest-load:
+    image: ghcr.io/parseablehq/quest:main
+    command: ["load", "http://parseable:8000", "parseableadmin", "parseableadmin", "20"]
+    networks:
+      - parseable-internal
+    depends_on:
+      quest-smoke: 
+        condition: service_completed_successfully
+      - parseable
```
Comment (nitisht) [inline docker-compose.yaml:82]: This doesn't look like it is needed here?

Comment (Sajiyah-Salat) [reply docker-compose.yaml:82]: are you talking about paraseable

Comment (Sajiyah-Salat) [reply docker-compose.yaml:82]: should I remove this.

Comment (Sajiyah-Salat) [reply docker-compose.yaml:82]: should I remove this.

Comment (nitisht) [reply docker-compose.yaml:82]: yes, please remove

Comment (Sajiyah-Salat) [reply docker-compose.yaml]: naming issues occur. What name should I give now?

**Comment by Sajiyah-Salat** [comment]
Finally! Thank you for helping me thorughout.

**Comment by Sajiyah-Salat** [comment]
I think before merge we should sqaush the commits.

**Comment by nitisht** [comment]
It's all squashed.

**Pull request merged**